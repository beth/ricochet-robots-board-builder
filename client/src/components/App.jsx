import React from 'react';
import Board from './Board';

const App = () => (
  <div>
    <Board size={8} />
  </div>
);

export default App;

