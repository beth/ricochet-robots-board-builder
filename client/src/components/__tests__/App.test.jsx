import React from 'react';
import renderer from 'react-test-renderer';
import App from '../App';

jest.mock('../Board', () => 'Board');

describe('App Component', () => {
  it('renders correctly', () => {
    const domTree = renderer.create(<App />).toJSON();
    expect(domTree).toMatchSnapshot();
  });
});
