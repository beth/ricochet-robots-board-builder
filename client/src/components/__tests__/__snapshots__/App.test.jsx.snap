// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component renders correctly 1`] = `
<div>
  <Board
    size={8}
  />
</div>
`;
